### DISCUSSION TITLE
hexo一键部署到github时的问题

### DISCUSSION THREAD

**Discussion by peixingzhe** [hexojs/hexo#5195] [Question / Answer] [answered]
根据官方文档，已经在`_config.yml`中配置好了git仓库的信息（包括token），但是每次执行`hexo d`的时候，还是提示输入账号密码，且当我输入正确的密码（token）时还是提示验证失败。
这个问题已经搞了好几天了还是解决不了，麻烦来个大手子帮忙看看。
![image](https://user-images.githubusercontent.com/46043280/234241134-b8ae6fb1-71e9-4af4-9b61-738ec3c29a98.png)
![image](https://user-images.githubusercontent.com/46043280/234241403-d46dbfe8-71c4-495c-b888-7bd6efba4857.png)

**Comment by yoshinorin** [comment]
@PeiMouRen 
How about https://github.com/hexojs/hexo-deployer-git/issues/159 ?

**Comment by nagongze** [comment]
<img width="410" alt="CleanShot 2023-04-26 at 07 55 45" src="https://user-images.githubusercontent.com/68653451/234432567-a852231a-9257-4080-95c2-0ebf9b7e0cf1.png">
試試看這樣？

**Comment by peixingzhe** [comment, marked as answer]
问题已解决。
升级最新版hexo，重新生成github token
